app/presenters/details_presenter.rb

```diff
@@ -2,23 +2,30 @@
 
 module Presenters
   class DetailsPresenter
-    attr_reader :content_item_details
+    attr_reader :content_item_details, :change_history_presenter
 
-    def initialize(content_item_details)
+    def initialize(content_item_details, change_history_presenter)
       @content_item_details = SymbolizeJSON.symbolize(content_item_details)
+      @change_history_presenter = change_history_presenter
     end
 
     def details
       @_details ||=
         begin
-          content_item_details.each_with_object({}) do |(key, val), seed|
+          updated = content_item_details.each_with_object({}) do |(key, val), seed|
             seed[key] = append_transformed_govspeak(val)
           end
+          updated[:change_history] = change_history unless change_history.blank?
```

Comment:
This is making two method calls, which in some case will mean two database queries for the same data.

I think it would be preferable to inline the `change_history` method, and use a variable, which gets rid of a entire method, and means improves readability (as well as removing the duplicate calls/database queries) e.g.:
```
change_history = change_history_presenter.change_history

updated[:change_history] = change_history unless change_history.blank?
```